geometry_msgs/Transform end_trans
bool                    is_done

---
float64[]             end2cam_trans

